docs/index.rst

```diff
@@ -1,53 +1,211 @@
+Which services are for me?
+==========================
 
-If you need any of those services for your machine, ``pro`` is the right tool
-for you. 
+Ubuntu Pro is a broad subscription designed to meet many different needs. You
+are unlikely to want to use all of these tools and services at the same time,
+so you are free to select the precise stream of updates and versions you want
+to apply on any given machine covered by your Pro subscription.
 
-``pro`` is already installed on every Ubuntu system. Try it out by running
-``pro help``!
+To help you navigate the various services offered through Ubuntu Pro, click on
+the tab that best represents your situation for our suggestions on which
+services might be of interest to you. For example, Pro includes a set of
+package versions that are compliant with FIPS regulations. You will only want
+these versions on machines that *need* to meet FIPS requirements, so you can
+choose to enable that stream specifically on those machines.
 
------
+.. tab-set::
+
+    .. tab-item:: Commercial or enterprise
+
+       * **Expanded Security Maintenance (ESM)**
+
+         If you're using an Ubuntu LTS -- Desktop or Server -- in a commercial
+         setting and don't want the inconvenience and downtime of upgrading to
+         a newer LTS yet, you can get additional security support through
+         `Expanded Security Maintenance (ESM)`_. We recommend it for everyone
+         using Pro, and it's enabled by default. It includes two services:
+
+         * ``esm-infra`` provides security updates for packages in the Ubuntu
+           'main' repository (packages from Ubuntu) for five additional years
+           beyond the standard five years of security maintenance of 'main'
+           packages.
+
+         * ``esm-apps`` covers packages in the 'universe' repository (which
+           are from Debian and the community) for ten years after the LTS
+           release. 
+
+         * Whether you need one or both of these depends on your subscription.
+           If you're not sure which services best support the needs of your
+           organisation, contact us for `a security assessment`_.
+
+       * **Livepatch**
+
+         Ubuntu `Livepatch`_ reduces costly unplanned maintenance by patching
+         vulnerabilities while the system runs. It removes the need to
+         immediately reboot the kernel after an upgrade so that you can
+         schedule downtime for when it's convenient. This tooling is available
+         through ``livepatch``.
+
+         * Find out more about what Livepatch can do for your organisation in
+           the `Livepatch documentation`_, or find out
+           :doc:`how to enable it<howtoguides/enable_livepatch>`.
+
+       * **Landscape** (coming soon!)
+
+         `Landscape`_
+
+       Want more information?
+
+    .. tab-item:: Certification and hardening
+
+       Ubuntu provides security compliance, `certifications, and hardening`_:
+
+       * **FIPS 140-2**
```

Comment:
I think that since we're not pointing to Landscape because it's not ready yet, we should probably hold off on this for the same reason. At least once it *is* certified we can include a link for more info. Wdyt?

Reply:
I agree!